.gitignore

```diff
@@ -1 +1,2 @@
 target
+/.vscode/
```

Comment:
I think this red symbol means there is a newline missing at the end of the file.